CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/Screens/CaptureScreen.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CaptureScreen.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+public class CaptureScreen {
+    let app: XCUIApplication
+    let captureButton: XCUIElement
+    let flashButton: XCUIElement
+    let filesUploadButton: XCUIElement
+    let cameraTitleText: XCUIElement
+    let cancelButton: XCUIElement
+    let helpButton: XCUIElement
+    let scanTextTitle: XCUIElement
+    
+    public init(app: XCUIApplication) {
+        self.app = app
+        captureButton = app.buttons[CaptureScreenAccessibilityIdentifier.captureButton.rawValue]
+        flashButton = app.buttons[CaptureScreenAccessibilityIdentifier.flashButton.rawValue]
+        filesUploadButton = app.buttons[CaptureScreenAccessibilityIdentifier.filesUploadButton.rawValue]
+        cameraTitleText = app.buttons[CaptureScreenAccessibilityIdentifier.cameraTitleText.rawValue]
+        cancelButton = app.buttons["Cancel"]
+        //cancelButton = app.buttons[CaptureScreenAccessibilityIdentifier.cancelButton.rawValue]
+        helpButton = app.buttons[CaptureScreenAccessibilityIdentifier.helpButton.rawValue]
+        scanTextTitle = app.buttons[CaptureScreenAccessibilityIdentifier.scanTextTitle.rawValue]
+    }
+    
+    public func assertFlashButton(){
```

Comment:
[suggestion] Can you please add space in front of `{`? 
Please check everywhere because I've seen it in multiple places.